texthero/preprocessing.py

```diff
@@ -14,16 +14,17 @@
 from nltk.stem import PorterStemmer, SnowballStemmer
 
 from texthero import stopwords as _stopwords
+from texthero._helper import TokenSeries, TextSeries, InputSeries
 
-from typing import List, Callable
+from typing import List, Callable, Union
 
 # Ignore gensim annoying warnings
 import warnings
 
 warnings.filterwarnings(action="ignore", category=UserWarning, module="gensim")
 
 
```

Comment:
Missing `@InputSeries(TextSeries)`?

Reply:
I believe you mean it's missing for `fillna`. If yes, just added it